### DISCUSSION TITLE
[Feature Request] Can we use the ‘.’ key to refresh the feeds when in slow mode?

### DISCUSSION THREAD

**Discussion by gurupanguji** [mastodon/mastodon#20190] [General]
First off, 

Really exciting to submit my first feature request. I saw that in the keyboard shortcuts, we currently don’t support a keyboard shortcut to refresh the feeds available “in view” when in slow mode. I propose we could / should use the ‘.’ (Period) key for the feature. 

This is something that’s common across other VIM keybindings are used too. Thanks.

**Comment by MrGeneration** [comment]
Hey don't wanna be rude but I believe the place where you want to file your enhancement suggestion would be here:
https://github.com/mastodon/mastodon/issues/new?assignees=&labels=suggestion&template=2.feature_request.yml

**Reply by gurupanguji** [reply]
Not rude at all and thanks for the pointer. Closing this out.

**Comment by ljantzen** [comment]
I submitted this request.  https://github.com/mastodon/mastodon/issues/23660
